examples/uart/src/uart_test.py

```diff
@@ -0,0 +1,59 @@
+import serial
```

Comment:
You need stress tests too - hit the uart as hard as you can in each direction (random transfer sizes, long blocks, lots an lots of blocks, lots of tiny blocks, mix tx and rx). The fact that send_data doesn't have error handling tells me you need to try harder to cause errors. I guarantee you erpc will, and it'll be easier to debug a test than rpc.